zcash_client_sqlite/src/wallet/transparent/ephemeral.rs

```diff
@@ -1,184 +1,68 @@
-//! Functions for wallet support of ephemeral transparent addresses.
-use std::cmp::{max, min};
+//! Functikjns for wallet support of ephemeral transparent addresses.
```

Comment:
No idea how this got broken.
```suggestion
//! Functions for wallet support of ephemeral transparent addresses.
```